### DISCUSSION TITLE
Stylesheet being loaded but not applied?

### DISCUSSION THREAD

**Discussion by RoRoo1977** [poweradmin/poweradmin#508] [Q&A]
I've installed poweradmin on my server and it works perfectly, but after changing to the spark stylesheet the dark mode is not activated. 
when I check the source, it lists the spark css and I can click through and read it, but it isn't applied? 

No errors in my httpd logs to indicate a problem. 

I don't believe it's a bug, but cannot seem to find the problem.

**Comment by edmondas** [comment]
@RoRoo1977 That sounds strange. What do you see when you change the theme in the config file? Are you seeing a page with broken styling or a light (default) theme instead? Make sure stylesheets are loading in your browser's network tab. Does your installation take place under a custom domain/subdomain or under a custom folder?

**Reply by RoRoo1977** [reply]
Yeah I found it strange as well. The theme file is included (I can see it defined in the source and can access it), but the look is the default stylesheet (light). It does not look wrong like no stylesheet is loaded. 

I'm using a custom domain name with it's own folder. 
Files/Directories are 644, owned by the apache user, running PHP-FPM 8.2 and MySQL backend. 
OS is CentOS 7
SSL is activated. 

Things worked on the older version with a custom css. 
When I installed the latest version, I moved the older versions directory so there are no conflicting files. 

I thought it could've been browser related or adblocker related but even a browser without addons shows the light css instead of the dark. 
Then I thought, well maybe it doesn't work because it's loaded after the bootstrap.min.css file and moved it to be loaded earlier, but that didn't work. 
I also experienced memory errors after enabling custom templates, so maybe it's just my server ;) 

I'll try another reinstall of PowerAdmin (and/or apache) and keep you updated.

**Comment by edmondas** [comment]
I'm closing this issue as it should no longer occur after multiple improvements to the style loading process. The problem may have been real when reported, but subsequent changes should have addressed it. If you encounter any new issues with style loading, please feel free to open a new issue. Thanks for your patience and contribution!
